docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -119,4 +119,4 @@ csc and vbc inside it. You can check the cibuild.cmd and see how it is used.
 
 ## Contributing
 
-Please see [Contributing Code](https://github.com/dotnet/roslyn/wiki/Contributing-Code) for details on contributing changes back to the code.
+Please see [Contributing Code](https://github.com/dotnet/roslyn/blob/master/CONTRIBUTING.md)) for details on contributing changes back to the code.
```

Comment:
Ditto to earlier question: is it possible to make this branch-relative so if you're in one branch it doesn't jump between?